examples/project_task_management/main.bal

```diff
@@ -0,0 +1,288 @@
+import ballerina/http;
+import ballerina/lang.'string as strings;
+import ballerina/log;
+import ballerina/regex;
+import ballerina/time;
+import ballerinax/slack;
+import ballerinax/smartsheet;
+
+//configurations
+configurable string smartsheetToken = ?;
+configurable string projectsSheetName = ?;
+configurable string tasksSheetName = ?;
+
+configurable string slackToken = ?;
+configurable string slackChannel = ?;
+
+// Column titles
+const string COL_PROJ_NAME = "Project Name";
```

Comment:
Built-in typed constants does not need to be defined with type. The type will be inferred automatically:
```suggestion
const COL_PROJ_NAME = "Project Name";
```